worlds/yoshisisland/docs/setup_en.md

```diff
@@ -0,0 +1,122 @@
+# Yoshi's Island Archipelago Randomizer Setup Guide
+
+## Required Software
+
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). Make sure to check the box for `SNI Client - Yoshi's Island Patch Setup`
+
+
+- Hardware or software capable of loading and playing SNES ROM files
+    - An emulator capable of connecting to SNI such as:
+        - snes9x-rr from: [snes9x rr](https://github.com/gocha/snes9x-rr/releases),
+        - BizHawk from: [TASVideos](https://tasvideos.org/BizHawk)
+        NOTE: RetroArch and FXPakPro are not currently supported.
+- Your legally obtained Yoshi's Island English 1.0 ROM file, probably named `Super Mario World 2 - Yoshi's Island (U).sfc`
+
+
+## Installation Procedures
+
+### Windows Setup
+
+1. During the installation of Archipelago, you will have been asked to install the SNI Client. If you did not do this,
+   or you are on an older version, you may run the installer again to install the SNI Client.
+2. During setup, you will be asked to locate your base ROM file. This is your Yoshi's Island ROM file.
```

Comment:
```suggestion
1. Download and install Archipelago from the link above, making sure to install the most recent version.
2. During generation/patching, you will be asked to locate your base ROM file. This is your Yoshi's Island ROM file.
```
Took these from the KDL3 docs, SNIClient is always included now and rom won't be asked for until you go to generate or patch.